webui/src/components/editrecord.jsx

```diff
@@ -168,7 +168,14 @@ const EditRecord = React.createClass({
             }
         }
         console.assert(!Array.isArray(value));
-        r = value !== undefined ? r.setIn(path, value) : r.deleteIn(path);
+        let trimmedValue;
+        if(typeof value === 'string' || value instanceof String) {
+            trimmedValue = value.replace(/^\s+/, '').replace(/\s+$/, ' ') ;
```

Comment:
let's use trim() for this (https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/String/trim)